spec/features/user_features_spec.rb

```diff
@@ -0,0 +1,18 @@
+require './app.rb'
+require 'spec_helper'
+require 'pg'
+
+feature 'peeps are displayed in reverse chronological order' do
+  scenario 'the first peep returned is the most recently added' do
+    connection = PG.connect :dbname => 'chitter_test'
+    visit '/'
+    fill_in 'enter_chitter', with: "Hello chitter world"
+    click_on 'Chit'
+    visit '/'
+    fill_in 'enter_chitter', with: "This should be first"
+    click_on 'Chit'
+    expect("This should be first").to appear_before("Hello chitter world")
```

Comment:
good feature test, very clear - however, it's failing on mine - not sure the timestamp feature is working, as it's returning an empty string for the time column